schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -49,6 +49,16 @@
           "meta:titleId": "aep-hygiene-ops##xdm:workorderID##title##7131",
           "meta:descriptionId": "aep-hygiene-ops##xdm:workorderID##description##50431"
         },
+        "xdm:targetProducts": {
```

Comment:
I would suggest to rename this to targetServices as it is more intuitive of what we mean here.

Reply:
Perhaps that would make sense for some teams, but it would confuse my team. We use consistently use "product" throughout our code, documentation, and JSON API's to refer to the various product teams we work with, such as `identity`, `profile`, and `datalake`.

Reply:
After consulting with my team, we've decided to accept your suggestion.